src/snowcli/cli/streamlit/commands.py

```diff
@@ -160,22 +138,29 @@ def streamlit_deploy(
     not exist.
     You can modify the behaviour using flags. For details check help information.
     """
-    if environment_file and not environment_file.exists():
+    dm = DefinitionManager()
+    streamlit = dm.project_definition.get("streamlit")
+    if not streamlit:
+        return MessageResult("No streamlit were specified in project definition.")
```

Comment:
Does this mean you *must* use a project definition, and cannot use cli arguments? I would assume that you could always do both.

Reply:
Yes, project definition will be the only way.